src/Navigation/WPF/Navigation.WPF/Models/Configuration.cs

```diff
@@ -1,21 +1,7 @@
-﻿using System;
-using System.Collections.Generic;
-
-namespace CodeMonkeys.Navigation.WPF
+﻿namespace CodeMonkeys.Navigation.WPF
 {
-    public class Configuration
+    public class Configuration :
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Change the name of type 'Configuration' to be different from an existing framework namespace.](https://app.codacy.com/gh/UltimateCodeMonkeys/CodeMonkeysMVVM/pullRequest?prid=5575746)